modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/AdminTooltip.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayAutocomplete from '@clayui/autocomplete';
+import ClayButton from '@clayui/button';
+import ClayCard from '@clayui/card';
+import {useResource} from '@clayui/data-provider';
+import ClayDropDown from '@clayui/drop-down';
+import ClayForm, {ClayInput, ClayRadio, ClayRadioGroup} from '@clayui/form';
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+
+const AdminTooltip = ({
+	namespace,
+	setRemovePinHandler,
+	setShowTooltip,
+	showTooltip,
```

Comment:
You only use the details, right?

Reply:
re are currently rethinking the whole implementation of this component. Please let me merge this because maybe we should remove as-is in the next [pr](https://issues.liferay.com/browse/COMMERCE-5797) replacing it in LR

Reply:
Yep, go ahead, nothing I suggested is blocking.